cli/internal/cloudcmd/create.go

```diff
@@ -18,6 +18,7 @@ import (
 	"runtime"
 	"strings"
 
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/to"
```

Comment:
```suggestion
```

nit: since this is CSP-agnostic code, it's a little bit cleaner to not import an Azure specific package here.
Since we have generics, you can simply define this at the bottom of the file (or in a separate file of the same package):

```go
func toPtr[T any](v T) *T {
	return &v
}
```